### ISSUE TITLE
Stop new users from posting spam on their userpages

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#4664]
**[Submitted to the original trac issue database at 12.59pm, Sunday, 4th November 2012]**

Find a way to stop new users from posting spam to their userpages. Currently 10-30%(varies from hour to hour) of new users only post spam to their userpages. 
Some stats: http://web.zone.ee/rmihkel87/ustats.html

I know that there is no solution that is easy and satifies everyone but we have to get rid of spamers.
Also most users change their description so rarely that they don't have anything against some kind of verification process.

Maybe require user to fill a captcha when user description contains links?

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 1.12pm, Sunday, 4th November 2012]**

We already have extensive protections in this area and there are hundreds of new users that you never see because those protections suspend the accounts as soon as they post the descriptions.

These spammers (who only post descriptions) are fairly benign as most people will never see them - the ones who post diary entries are a bigger problem.

I have reason to believe that most of the spammers are real humans rather than robots so captchas achieve very little, except to reduce our conversion rate for real users.

In short this is something we already dealing with all the time and I don't believe there is anything more we can do at the moment that would help.

**Comment by tomhughes** [comment]
**Author: RM87**
**[Added to the original trac issue at 2.00pm, Sunday, 4th November 2012]**

I know very well that it is easier to ignore the problem than to deal with it. 

Assuming that most of the users about who api returns an empty page are users that these "extensive protections" have caught. In that case these measures only catch less than halve of spam users. See the last lines of: http://web.zone.ee/rmihkel87/ustats.html 

I doubt that these spammers are beginners and that their intention is to directly feed these links to humans. Rather they play on the fact that google and other serch engines rise the rating of their selected page if search engine bots find a link to it.

Looking at the usernames of current spammers that escape current measures, we can see that these are made of 2 english words and 1-3 numbers. Either 99% of current spammers have made an alliance to use same schema for usernames or there is just one bot.

And sorry for being a little bit harch.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 2.21pm, Sunday, 4th November 2012]**

Look unless you have an actual concrete solution for something nobody else in the world has managed to solve (getting rid of spammers) then I really don't see the point in opening a bug that can never be fixed.
